numpy/testing/_private/utils.py

```diff
@@ -19,6 +19,7 @@
 from warnings import WarningMessage
 import pprint
 
+import numpy as np
```

Comment:
In other places in this file, the `import numpy as np` has been moved into function scope. I suggest we use the same approach for your changes.